### PR TITLE
Understanding graphql introspection system

### PR DESCRIPTION
**Pull request by shegz101**
This post will provide a comprehensive guide to understanding the GraphQL introspection system. We will examine its major concepts.

### CODE REVIEW

File: outlines/exploring-graphiql-the-in-browser-ide-for-graphql.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+* Fragments: This feature helps us maintain the Don’t Repeat Yourself (DRY) pattern when dealing with repetitive code in our query. Instead, you can easily merge fragments into a query.
+```
+# code that references fragments that can be merged to a query
+```
+
+* Documentation: GraphiQL generates documentation for your schema, making it easy to understand and explore.
+
+[Screenshot the documentation interface]
+
+## Testing out the Interface
```
Comment (amitksingh1490) [inline outlines/exploring-graphiql-the-in-browser-ide-for-graphql.md]: Heading is confusing

File: outlines/debugging-graphql-queries-with-graphiql.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+- Understanding what debugging is and its importance concerning GraphQL queries
+- Why is GraphiQl a good option for debugging GraphQl queries (query validation and error handling)
+
+## Instances that can lead to bugs in the Queries
+- Identifying the common errors that can generate bugs in a query. Highlight and explain them.
+- The common errors are Syntax, Type Mismatches, and Field Resolution issues.
+
+[analyze each of these errors and add a screenshot sample ]
+
+## Practical Scenario of Debugging a GraphQL Query in GraphiQL
```
Comment (amitksingh1490) [inline outlines/debugging-graphql-queries-with-graphiql.md]: I think this heading is not needed.

File: outlines/debugging-graphql-queries-with-graphiql.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
+# Debugging GraphQl queries with GraphiQL
```
Comment (amitksingh1490) [inline outlines/debugging-graphql-queries-with-graphiql.md]: This whole blog could be merged with 
Exploring GraphiQL: The In-Browser IDE for GraphQL

File: outlines/understanding-graphql-introspection-system.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
+# Understanding GraphQL's Introspection System
```
Comment (amitksingh1490) [inline outlines/understanding-graphql-introspection-system.md:1]: How is this different than what we already have here https://tailcall.run/graphql/graphql-introspection/

**Comment by Frenzyritz13** [comment]
Hey @shegz101, if you are still working on this, I would recommend using this checklist to review your article: https://docs.google.com/document/d/1Bw9iT4ws9q093kCJsnStfpByJljn-wRPNtc575s3_R8/edit?usp=sharing
And if you want discuss changes and flow of it feel free to put some time on my calendar here: https://calendly.com/ritvi

**Comment by shegz101** [comment]
@amitksingh1490 should this PR be closed?

**Pull request closed without merging**